docs/reference/core_v3.rst

```diff
@@ -0,0 +1,122 @@
+-----------
+Core API v3
+-----------
+
+.. py:currentmodule:: imageio.core.v3_api
+
+.. note::
+    For migration instructions from the v2 API check :ref:`our migration guide
+    <migration_from_v2>`.
+
+This API provides a functional interface to all of ImageIOs backends. It is
+split into two parts: :ref:`Everyday Usage <v3_basic_usage>` and :ref:`Advanced
+Usage <v3_advanced_usage>`. The first part (everyday usage) focusses on sensible
+defaults and automation to cover everyday tasks such as plain reading or writing
+of images. The second part (advanced usage) focusses on providing fully-featured
+and performant access to all the a particular backend.
+
+API Overview
+------------
+
+.. autosummary::
+    :toctree: _core_v3/
+
+    imageio.core.v3_api.imread
+    imageio.core.v3_api.imwrite
+    imageio.core.v3_api.imiter
+    imageio.core.v3_api.imopen
+
+.. _v3_basic_usage:
+
+Everyday Usage
+--------------
+
+Most of the time, the image-related IO we wish to perform can be described as "I
+want to load X as a numpy array.", or "I want to save my array as a XYZ file".
+This should not be complicated, and it is, in fact, as simple as calling the
+corresponding function::
+
+    import imageio as iio
+
+    # reading:
+    img = iio.v3.imread("imageio:chelsea.png")
+    
+    # writing:
+    iio.v3.imwrite("out_image.jpg", img)
+
+Both functions accept any :doc:`ImageResource <../getting_started/requests>` as
+their first argument, which means we can read/write almost anything image
+related; from simple JPGs through ImageJ hyperstacks to MP4 video or various
+niche RAW formats.
+
+If we do need to customize or tweak this process we can pass additional keyword
+arguments (`kwargs`) to overwrite any defaults ImageIO uses when reading or
+writing. The two `kwargs` that are always available are:
+
+- ``plugin``: The name of the plugin to use for reading/writing.
+- ``index`` (reading only): The index of the ndimage to read, if the format
+  supports storing more than one (GIF, multi-page TIFF, video, etc.).
+
+Additional `kwargs` are specific to the plugin being used and we can find a list
```

Comment:
```suggestion
Additional `kwargs` are specific to the plugin being used and you can find a list
```

Reply:
Hm, do we want to use `we`, `you`, or passive tense in the docs? 

I tend to mix and match `we` and `you`, but should be consistent with one or the other. Personally, I dislike the "old academic style" of putting passive tense everywhere.

Reply:
The passive tense can indeed become a bit "artificial". Otherwise it's a somewhat personal choice, but I find "we" a bit odd in docs like this.

Reply:
Sure, then I will change the docs to use "you".